utils/json/src/signature.rs

```diff
@@ -119,46 +119,21 @@ mod tests {
     use quickcheck_macros::quickcheck;
     use serde_json;
 
-    use super::json::{signature_type::SignatureTypeJson, SignatureJson, SignatureJsonRef};
-
-    #[derive(Clone, Debug, PartialEq)]
-    struct SignatureWrapper {
-        signature: Signature,
-    }
-
-    #[derive(Clone, Debug, PartialEq)]
-    struct SignatureTypeWrapper {
-        sigtype: SignatureType,
-    }
-
-    impl quickcheck::Arbitrary for SignatureWrapper {
-        fn arbitrary(g: &mut quickcheck::Gen) -> Self {
-            let sigtype = SignatureTypeWrapper::arbitrary(g);
-            let signature = Signature::new(sigtype.sigtype, Vec::arbitrary(g));
-            SignatureWrapper { signature }
-        }
-    }
-
-    impl quickcheck::Arbitrary for SignatureTypeWrapper {
-        fn arbitrary(g: &mut quickcheck::Gen) -> Self {
-            let sigtype = g
-                .choose(&[SignatureType::Secp256k1, SignatureType::BLS])
-                .unwrap();
-            SignatureTypeWrapper { sigtype: *sigtype }
-        }
-    }
-
     #[quickcheck]
-    fn signature_roundtrip(signature: SignatureWrapper) {
-        let serialized = serde_json::to_string(&SignatureJsonRef(&signature.signature)).unwrap();
-        let parsed: SignatureJson = serde_json::from_str(&serialized).unwrap();
-        assert_eq!(signature.signature, parsed.0);
+    fn signature_roundtrip(signature: Signature) {
+        let serialized = forest_test_utils::to_string_with!(&signature, super::json::serialize);
+        let parsed = forest_test_utils::from_str_with!(&serialized, super::json::deserialize);
```

Comment:
Why switch to `to_string_with!` rather than using `SignatureJsonRef`?

Reply:
I thought we had preferred to use the `forest_test_utils` macros in previous `quickcheck` tests, but perhaps I'm mistaken. I would argue that it improves readability a bit, but I can change it back to using `SignatureJsonRef` if that's better.

Reply:
The macros are there to assist if you do not have a helper structure such as `SignatureJsonRef`.